File: challenges/2-tabuada/c/paclucas/tabuada.c
#include <stdio.h>

int  main(){
  int amt;

  printf("Deseja a tabuada até qual número?: ");
  scanf("%d",&amt);

  for(int i = 1; i <= amt; i++){
    printf("=~~~~~~~~~~~~~~~~~~~~~~~~~=");
    printf("\nA tabuada  do  número %d é:\n", i);
    
    for(int j = 1; j <= 10; j++)  printf("%d * %d = %d\n", i, j, i * j);
  }
  printf("=~~~~~~~~~~~~~~~~~~~~~~~~~=");

  printf("\n^._.^\n");

  return 0;
}

File: challenges/2-tabuada/python/jbernanos/tabuada.py
def dados():
    numero = int(input('Informe até qual número deseja ver a tabuada: '))
    tabuada(numero)
    return numero

def tabuada(numero):
    for i in range(1, numero + 1):
        print(' ')
        print(f'A tabuada do numero {i} é:')
        for k in range(1, 11, 1):
            calculo = i * k
            print(f'{i} x {k} = {calculo}')

dados()
